base/error.go

```diff
@@ -31,12 +32,15 @@ type SGError struct {
 
 var (
 	ErrAlreadyImported = &SGError{alreadyImported}
+	ErrImportCancelled = &SGError{importCancelled}
 )
 
 func (e SGError) Error() string {
 	switch e.code {
 	case alreadyImported:
 		return "Document already imported"
+	case importCancelled:
```

Comment:
Ok, after looking at the later code, I see that `ImportCancelled` means something different than my first comment, but I still think it's worth documenting a bit more so that someone reading the logs or code will know more about why this happens and whether they need to do anything about it.